litestar/middleware/exceptions/_debug_response.py

```diff
@@ -48,8 +48,13 @@ def get_symbol_name(frame: FrameInfo) -> str:
     locals_dict = frame.frame.f_locals
     # this piece assumes that the code uses standard names "self" and "cls"
     # in instance and class methods
-    instance_or_cls = locals_dict.get("self") or locals_dict.get("cls")
-    classname = f"{get_name(instance_or_cls)}." if instance_or_cls else ""
+    instance_or_cls = None
+    if "self" in locals_dict:
+        instance_or_cls = locals_dict["self"]
+    elif "cls" in locals_dict:
+        instance_or_cls = locals_dict["cls"]
```

Comment:
```suggestion
    instance_or_cls = inst if (inst := locals_dict.get("self")) is not None else locals_dict.get("cls")
```
Can still be done in compact form - just a nit-pick though, these are functionally the same.